### PR TITLE
elasticsearch6 driver bulkCommit implementation

### PR DESCRIPTION
**Pull request by nanov**
A basic bulkCommit implementation.

Additional things that can be done: 

*. By huge amount of updates / vms ( ex: 500+, configurable of course) at once, the bulk operation can be split into smaller ones.

*. Introduction of BulkCommitError which is a container of multiple errors and can be feed it with individual errors for each vm ( will need some a little rework from the other implementations ).

### CODE REVIEW

File: lib/databases/elasticsearch6.js [hunk 1]
```diff
@@ -150,10 +154,14 @@ _.extend(Elasticsearch.prototype, {
 				},
 				function(err, res) {
 					if (err) {
-						return callback(err);
+            console.log('---')
```
Comment (adrai) [inline lib/databases/elasticsearch6.js]: probably not needed, right? ;-)

Comment (nanov) [reply lib/databases/elasticsearch6.js]: damn :)

**Comment by nanov** [comment]
hmm, for some reason mocha hangs after finish, any ideas?

File: lib/databases/elasticsearch6.js [hunk 2]
```diff
@@ -170,20 +178,115 @@ _.extend(Elasticsearch.prototype, {
...
+          case 'delete':
+            if (vm.version === null) return;
+            bulkOperation.push({ delete: { _index: self.indexAndTypeName, _type: self.indexAndTypeName, _id: vm.id, version: vm.version } });
+            deletedCount++;
+            break;
+          case 'create':
+            obj = vm.attributes;
+            bulkOperation.push(
+              { index: { _index: self.indexAndTypeName, _type: self.indexAndTypeName, _id: vm.id, op_type: 'create' } },
+              obj,
```
Comment (adrai) [inline lib/databases/elasticsearch6.js]: is there a missin } ?

Comment (nanov) [reply lib/databases/elasticsearch6.js]: not really, but there is a trailing comma which is a problem on older node

**Comment by nanov** [comment]
nailed it ;)

**Comment by adrai** [comment]
nice job!

**Pull request merged**